src/_nested-list.scss

```diff
@@ -0,0 +1,307 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-nested-list;
+
+@mixin fd-side-nav-arrow {
```

Comment:
nit: since this is `nested-list.scss`, this is probably better named `fd-nested-list-arrow` to not couple to the "side-nav" concept